pages/blog/how-not-to-learn-graphql.mdx

```diff
@@ -0,0 +1,776 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'How not to learn GraphQL',
+  author: 'charlypoly',
+  tags: ['graphql'],
+  date: '2022-02-14',
+  description:
+    'A guide to avoid common learning biases and misconceptions around GraphQL',
+  thumbnail: '/blog-assets/how-not-to-learn-graphql/preview.png',
+  image: '/blog-assets/how-not-to-learn-graphql/cover.png'
+}
+
+export default withArticle({ ...meta })
+
+"_GraphQL is a front-end technology_," "_Federation is the only solution for unified graph and, services orchestration_," "_GraphQL is a replacement of REST"_ are common misconceptions that are most likely linked to biased learning of GraphQL.
+
+Because of its rapid ecosystem evolution in the past years, learning GraphQL in 2022 is more challenging than it used to be.
+
+For this reason, this article, greatly inspired by "[How not to learn TypeScript](https://fettblog.eu/how-not-to-learn-typescript/)" and "[How not to learn Rust](https://dystroy.org/blog/how-not-to-learn-rust/)," will guide you to avoid common learning biases and misconceptions around GraphQL.
```

Comment:
```suggestion
For this reason, this article, greatly inspired by "[How not to learn TypeScript](https://fettblog.eu/how-not-to-learn-typescript/)" and "[How not to learn Rust](https://dystroy.org/blog/how-not-to-learn-rust/)", will guide you to avoid common learning biases and misconceptions around GraphQL.
```